main.css

```diff
@@ -294,34 +294,35 @@
     font-family: 'Meera Inimai', sans-serif;
     font-weight: 900;
     font-size: 32px;
+    text-align: center;
     color: #091e42;
   }
 
-  .contact-form-section .form-inputs {
-    margin-top: 20px;
+  .contact-form-section .form-inputs form {
+    margin-top: 40px;
     display: flex;
     flex-direction: column;
     gap: 18px;
     width: 100%;
   }
 
-  .contact-form-section .form-inputs input {
+  .contact-form-section .form-inputs form input {
     outline: 0;
     border-width: 0 0 2px;
     border-color: gray;
     padding-bottom: 30px;
   }
 
-  .contact-form-section .form-inputs textarea {
+  .contact-form-section .form-inputs form textarea {
     padding: 10px;
     font-size: 14px;
   }
 
-  .contact-form-section .form-inputs .getintouchbtn {
+  .contact-form-section .form-inputs form .getintouchbtn {
     background-color: #ff6b00;
     width: 80%;
     align-self: center;
-    padding: 20px;
+    padding: 16px;
```

Comment:
- [ ] Let's try to follow the design guidelines for our submit button. Please adjust the `width` , `font-size`, and `font-weight` to match the Figma template 🙏 
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/166437924-691012f7-ed9a-4ef7-ae98-24490b96a20f.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/166437972-05317993-c7bc-4737-9315-ab4b42a4d5d1.png)